src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/hmi/on_driver_distraction_notification.cc

```diff
@@ -126,6 +127,19 @@ void OnDriverDistractionNotification::Run() {
       static_cast<int32_t>(application_manager::MessageType::kNotification);
   (*on_driver_distraction)[strings::msg_params][mobile_notification::state] =
       state;
+  if ((*message_)[strings::msg_params].keyExists(
+          mobile_notification::lock_screen_dismissal_enabled)) {
+    const bool lock_screen_dismissal_enabled =
+        (*message_)[strings::msg_params]
+                   [mobile_notification::lock_screen_dismissal_enabled]
+                       .asBool();
+    application_manager_.set_lock_screen_dismissal_state(
+        static_cast<LockScreenDismissalEnabled>(lock_screen_dismissal_enabled));
```

Comment:
@IGapchuk avoid casting bool to enum. 
Use it condition and specify enum state explicitly.

Reply:
@LuxoftAKutsan  Fixed in cd078272812e56dbc954d3d9319ac822f24b3093.

Reply:
@IGapchuk not fixed